src/lib/whenIdleOrHidden.ts

```diff
@@ -29,7 +29,12 @@ export const whenIdleOrHidden = (cb: () => void) => {
     cb();
   } else {
     cb = runOnce(cb);
-    rIC(cb);
     document.addEventListener('visibilitychange', cb, {once: true});
+    rIC(() => {
+      cb();
+      // Remove the above event listener since no longer required.
+      // See: https://github.com/GoogleChrome/web-vitals/issues/622
+      document.removeEventListener('visibilitychange', cb);
```

Comment:
```suggestion
      document.removeEventListener('visibilitychange', cb, {once: true});
```
I think you might need to make the signature exactly the same in order for the listener ti get removed. Can you confirm?

Reply:
I just tried and it seems to work both ways, so LGTM.